requirements.txt

```diff
@@ -2,5 +2,5 @@ pyee
 requests
 qrcode
 lxml
-wechaty-puppet>=0.0.17
-wechaty-puppet-hostie>=0.2.14
+wechaty-puppet==0.0.19
```

Comment:
Could you please explain why we are going to use `==` instead of `>=` ?

From the Node.js practice, we are always using `=~` in our dependencies in `package.json`, which I guess should be equal to `>=` in `requirements.txt` in Python?

Reply:
Yes, we use `>=` to control version dependency before,  but it gets into trouble.  Description : 
- `python-wechaty-puppet-hostie` has the `>=` version dependency with `python-wechaty-puppet`, which means tha `python-wechaty` always install latest version of `python-wechaty-puppet`
- One day, we add a new feature to `python-wechaty-puppet`, eg: `dirty_payload` abstract method. At the this time, we have not completed `python-wechaty-puppet-hostie`, and the new customer will update the latest `python-wechaty-puppet` in `python-wechaty-puppet`, which will cause `abstract method not implemented` error. 

And this problem should be avoided by merging times. 

- the strict version dependency will make stable version control, which will make `python-wechaty` more stable. 

> This is my consideration. 

@huan  But `>=` may be more suitable to keep dependency latest to touch latest features. A very specific problem.

Reply:
Could you please read the SemVer.org and confirm the behavior of `>=` dependency?

In node, `=~` will use the latest patch version of the same minor version.

If we have any breaking changes, we should bump the minor version, so that the new version will not effect the old codes.

Reply:
After reading the document of [`semver.org`](https://semver.org/) and version dependency rules in [pypi](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers) .  These are some points we need to know:
- `>=` will get the latest version, which is the same mechanism as `npm`.
- `~=` in pypi will keep `major` same version, which the `minor` versio should get the latest.  eg: `pip install Flask~=0.8` command will install `0.12.5` version. and the `pip install Flask>=0.8` command will install `1.1.2`
- `pip` command is no more precision in version control than `npm`

So, to achieve more granular versioning as `npm`, we can use `>=0.6.0,<0.7` command to keep the same `minor` version, and the latest `patch` version. 

I guess, we can keep the latest `minor` version with command `>=major.minor, <major.minor+1` . @huan  how do you think about ?

Reply:
Merge to #150